test/rhsmlib/dbus/test_register.py

```diff
@@ -413,6 +421,26 @@ def test_Register(self):
         result = self.obj.Register.__wrapped__(self.obj, "org", "username", "password", {}, {}, self.LOCALE)
         self.assertEqual(expected, json.loads(result))
 
+    def test_Register__with_force_option(self):
+        expected = json.loads(CONSUMER_CONTENT_JSON)
+        self.patches["register"].return_value = expected
+        self.patches["unregister"].return_value = None
+        self.patches["is_registered"].return_value = True
+
+        result = self.obj.Register.__wrapped__(
+            self.obj, "org", "username", "password", {"force": "1"}, {}, self.LOCALE
```

Comment:
> {"force": "1"}

hm the `"1"` should be a proper boolean, as key is a "D-Bus boolean" which then it is converted to a Python boolean by dbus-puthon